backend/docker_test.go

```diff
@@ -265,6 +265,7 @@ func TestNewDockerProvider_WithInvalidCPUSetSize(t *testing.T) {
 	defer dockerTestTeardown()
 
 	assert.NotNil(t, err)
+	assert.Equal(t, "strconv.ParseInt: parsing \"fafafaf\": invalid syntax", err.Error())
```

Comment:
Is this a leftover from your other branch addressing cpusets?

Reply:
Yes indeed, my bad! I can take it out if you want, as it doesn't really belong in this PR, though I don't think it hurts either.

Reply:
I don't think it hurts 😃